src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -804,6 +809,42 @@ internal async void ExecutePackageDownload(string name, PackageVersion package,
                     if (res == MessageBoxResult.Cancel || res == MessageBoxResult.None) return;
                 }
 
+                // determinate if any of the packages are targeting other hosts
+                var containsPackagesThatTargetOtherHosts = false;
+
+
+                // Known hosts
+                var knownHosts = Model.GetKnownHosts();
+
+                // Sandbox, special case: Warn if any package targets only one known host
+                if (String.IsNullOrEmpty(Host))
```

Comment:
yeah, it's too bad the "Hosts" entry is not super well defined, and that core or sandbox does not exist. I think this logic will probably need to be flexible.

Reply:
Yep, a have tried to be conservative and avoid false positives in sandbox. Our generic packages tends to be marked as targeting no known hosts or all known hosts. A package is most likely a "works on this host only" package if it targets only one host.